assets/app/styles/_core.less

```diff
@@ -693,6 +693,9 @@ select:invalid {
   .text-overflow();
 }
 
+.word-break-all {
+  .word-break-all()
+}
```

Comment:
If you leave off the `()` when you define the style in _mixins.less, you can use it as a mixin or a normal class. Then you don't need the duplicate definition. Same comment for existing `.word-break` and `.word-break()`.

http://lesscss.org/features/#mixins-feature